walkthrough/05-aws/README.md

```diff
@@ -0,0 +1,14 @@
+# 05 - AWS
+
+It's a safe guess that most Terragrunt users use it to manage AWS infrastructure.
+
+Here we'll take a look at how Stacks can be used to manage AWS infrastructure in a more scalable way.
```

Comment:
/s/Many Terragrunt users use Terragrunt to manage AWS infrastructure,  so here we'll take a look...

Reply:
Do we actually know that to be true? I didn't.